<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="genCount">
      <property name="font">
       <font>
        <family>Old English Text MT</family>
        <pointsize>24</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Generation:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Start">
      <property name="text">
       <string>Next Gen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="autoPlay">
      <property name="text">
       <string>Autoplay</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="Reset">
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="gridVal">
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
